xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-api/src/main/java/org/xwiki/model/internal/reference/SymbolScheme.java

```diff
@@ -38,6 +39,15 @@
      */
     Character getEscapeSymbol();
 
+    /**
+     * @return the separator used to separate the entity type from the reference
+     * @since 14.8RC1
+     */
+    default Character getEntityTypeSeparator()
+    {
+        return ':';
```

Comment:
Is it the same separator as https://github.com/xwiki/xwiki-platform/pull/1891/files#diff-b03a9b20f9c0d2911ca9a8c6cfbcf08b227757ef36e05905dee9615b14a98ca7R92 ?

Reply:
It is the same character. I could indeed reuse the variable from DefaultSymbolScheme even if it might not be clean to have this reference to DefaultSymbolScheme in the interface.

Reply:
Ha, I remember now, I did not want to make it public like all the others variables in DefaultSymbolScheme.